### PR TITLE
Tableau unit tests + bugfix

### PR DESCRIPTION
**Pull request by mgray35**
- Added unit testing for Tableau (100% coverage).
- Fixed a bug in canMove() that allowed any card to be removed as long as an Ace exists on the board, instead of requiring that the card to be removed is an Ace.

### CODE REVIEW

**Comment by mgray35** [comment]
I am getting a 400 server error when trying to test this. It might be because the added parameter in canMove() is messing with the routes somehow, but I can't find the problem.

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -371,12 +371,12 @@ <h2 class="modal-title"></h2>
...
-                	setTimeout(function () {
-                    	    $(select).toggleClass("redBackground");
-                	}, 90);
-            	    }
+//		    if (game.feedbackText != "" && game.feedbackText != "Joker used!") {
+//                	$(select).toggleClass("redBackground");
+//                	setTimeout(function () {
+//                    	    $(select).toggleClass("redBackground");
+//                	}, 90);
+//            	    }
```
Comment (mgray35) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: It turns out this section is what was causing the 400 error. Commenting it out is just a temporary fix.

Comment (osugraytr) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: Should I push this??

**Comment by osugraytr** [comment]
Hey, was this to show is what is going on? Or should I push this?

**Comment by thadenf** [comment]
The code looks good as far as I can see. Thought I dont completely understand the language.

**Pull request merged**